bundle/manifests/collector_monitoring.coreos.com_v1_prometheusrule.yaml

```diff
@@ -61,6 +61,48 @@ spec:
       labels:
         service: collector
         severity: Warning
+    - alert: ElasticsearchDeprecation
+      annotations:
+        message: The OpenShift Elasticsearch Operator is deprecated and is planned
+          to be removed in a future release. Red Hat provides bug fixes and support
+          for this feature during the current release lifecycle, but this feature
+          no longer receives enhancements. As an alternative to using the OpenShift
+          Elasticsearch Operator to manage the default log storage, you can use the
+          Loki Operator.
+        summary: Detected Elasticsearch as the on cluster storage which is being deprecated
+          in a future release.
+      expr: |
+        sum(min_over_time(container_cpu_usage_seconds_total{container=~'elasticsearch'}[5m])) > 0
```

Comment:
We need to restrict this to "openshift-logging" NS since AFAIK tracing could still be relying upon 'elasticsearch'.  We may want to also see if there is a reliable label on the pod in lieu of container given it's possible maybe to have an Elastic deployed operator managing an elasticsearch cluster

Reply:
I've used metrics suggested by @xperimental below